test/remote/gateways/remote_mondido_test.rb

```diff
@@ -0,0 +1,1487 @@
+require 'test_helper'
+
+class RemoteMondidoTest < Test::Unit::TestCase
```

Comment:
Some tests require some scenarios that are specific so I've used my account. Was dynamically before but I was asked to remove in one of the previous comments. You can take the acc that I send you privetely for your tests then. Can you send me an email to anderson {at} mondido.com?